### PR TITLE
Category changed to Title in SFS list

### PR DESCRIPTION
**Pull request by plasticviking**
# Changelog

Change 'Category' to 'Title' in SFS list
The format is {title}: {milestone} for Part 3 Evidence submissions, {title} otherwise

Trello card 1161

### CODE REVIEW

**Comment by amichard** [comment]
I might have some submissions that don't have milestones, but I'm running into an error when I load this with the page. 

Can we move the logic in the front-end as well? And just add a check there? Just so the back-end isn't controlled by the requirements in the front-end

**Comment by plasticviking** [comment]
@amichard This one is updated too. The logic is now in the frontend and it handles the case where milestone is None.

File: frontend/src/secure_document_upload/components/CreditTransferRequestTable.js [hunk 1]
```diff
@@ -39,9 +39,19 @@ const CreditTransferRequestTable = (props) => {
     id: 'status',
     minWidth: 50
   }, {
-    className: 'col-category',
-    Header: 'Category',
-    id: 'category',
+    accessor: item => {
+      if (item.type.theType === 'Evidence') {
+        if (item.milestone !== null && item.milestone.milestone.length > 0) {
```
Comment (amichard) [inline frontend/src/secure_document_upload/components/CreditTransferRequestTable.js]: I seem to be getting an error with the check if the document evidence is missing a milestone:
```Cannot read property 'length' of null```

The check might need to be changed to:
```if (item.milestone && item.milestone.milestone) {```
or just
```if (item.milestone) {```

Comment (plasticviking) [reply frontend/src/secure_document_upload/components/CreditTransferRequestTable.js]: @amichard Fixed with the latest push

**Pull request merged**